dissect/target/plugins/os/unix/log/atop.py

```diff
@@ -0,0 +1,362 @@
+import logging
+import zlib
+from io import BytesIO
+from typing import BinaryIO, Iterator
+
+from dissect.cstruct import Instance, cstruct
+from flow.record.fieldtypes import path
+
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+log = logging.getLogger(__name__)
+
+atop_versions = ["2.6", "2.7"]
+atop_def = """
+typedef unsigned long long time_t;
```

Comment:
Could you please fix the alignment of the C-structs here and add the C-sources here instead of below?

Reply:
Do you mean that the C-structs should be saved in a single variable without `padding`?

Reply:
No that is not necessary, if you could just fix the alignment that would be really nice.